testing/vcs/test_vcs_set_line_attribute_from_line.py

```diff
@@ -6,16 +6,22 @@
 l.type = "dash"
 
 v=vcs.createvector()
-v.line = "vcs_test_set_line"
+v.setLineAttributes("vcs_test_set_line")
 
 assert(v.linecolor == 242)
 assert(v.linewidth == 5.6)
-assert(v.line == "dash")
+assert(v.linetype == "dash")
 
 yx = vcs.create1d()
-yx.line = l
-
+yx.setLineAttributes(l)
 
 assert(yx.linecolor == 242)
 assert(yx.linewidth == 5.6)
-assert(yx.line == "dash")
+assert(yx.linetype == "dash")
+
+iso = vcs.createisoline()
+iso.setLineAttributes([l, "solid", l])
```

Comment:
maybe "solid" isn't the best choice here, it could llead people to think they can pass a line `type` instead of a line name. I would use something very different like: `thick` or `pink`